contracts/true-currencies/TrueCurrencyWithPoR.sol

```diff
@@ -49,13 +49,11 @@ abstract contract TrueCurrencyWithPoR is TrueCurrency, IPoRToken {
         // Get required info about total supply & decimals
         uint8 trueDecimals = decimals();
         uint8 reserveDecimals = AggregatorV3Interface(chainReserveFeed).decimals();
+
+        // Decimals of the Proof of Reserve feed must be the same as the token's.
+        require(reserveDecimals == trueDecimals, "Unexpected decimals of PoR feed");
+
         uint256 currentSupply = totalSupply();
```

Comment:
Can we inline `totalSupply()` into the check below, now that the `currentSupply` variable is no longer used elsewhere?

Reply:
Done.